package.json

```diff
@@ -10,6 +10,7 @@
   "scripts": {
     "build": "npm run build:css && react-scripts build",
     "build:css": "node-sass src/ -o src/",
+    "postbuild":"rm -rf build/api",
```

Comment:
If we add this `postbuild` script, do we need to run `rm` manually?

Reply:
nope, it runs automatically when you npm run build, i tried it

Reply:
then can we not include the change in the `Makefile`? :D

Reply:
uhh i read this wrong... i thot u meant can't we do the above in makefile... hence the discussion. Sorry!